apps/src/applab/designMode.js

```diff
@@ -859,7 +859,7 @@ designMode.renderToggleRow = function (screenIds) {
   var designToggleRow = document.getElementById('designToggleRow');
   if (designToggleRow) {
     React.render(
-      React.createElement(DesignToggleRow, {
+      React.createElement(PlaySpaceHeader, {
```

Comment:
This comment isn't particular to this PR, and not something you necessarily need to address. But I think with our current build system we can using JSX in .js files. This means, we could get rid of the React.createElement here and do
```
<PlaySpaceHeader
  hideToggle: {Applab.hideDesignModeToggle()}
  ...
/>